### PR TITLE
Job

### PR DESCRIPTION
**Pull request by lukasz-ligizynski**
I think,  I could forget something important, or did the wrong naming. My problem right now looks like this:  
image
 I think the problem is with finding seance.

### CODE REVIEW

File: spec/requests/reservations_request_spec.rb [hunk 1]
```diff
@@ -22,19 +22,21 @@
   end
 
   describe 'POST /reservations' do
-    let(:seance) { create(:seance) }
-    let(:client) { create(:client) }
-    let(:ticket_desk) { create(:ticket_desk) }
+    let!(:seance) { create :seance }
```
Comment (wmaciejak) [inline spec/requests/reservations_request_spec.rb:25]: `create :seance, seance_date: DateTime.zone.now`

File: app/concepts/reservations/use_cases/create.rb [hunk 2]
```diff
@@ -10,7 +10,19 @@ def initialize(repository: Reservations::Repository.new)
       end
 
       def call(params:)
-        repository.create(params)
+        @reservation = repository.create(params)
+        cancel_reservation
```
Comment (wmaciejak) [inline app/concepts/reservations/use_cases/create.rb:14]: Consider this:

```
@reservation = repository.create(params)
cancel_reservation
@reservation
```
or 
```
@reservation = repository.create(params).tap { cancel_reservation }
```

Comment (lukasz-ligizynski) [reply app/concepts/reservations/use_cases/create.rb:14]: Done and working :D

File: spec/requests/reservations_request_spec.rb (hunk 1 above)
Comment (lukasz-ligizynski) [reply spec/requests/reservations_request_spec.rb:25]: Fixed -> there was no seance_date in factory

**Pull request merged**